Okta.Xamarin/Okta.Xamarin/AuthCodeTokenExchangeExceptionEventArgs.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Okta.Xamarin
+{
+    /// <summary>
+    /// Data relevant to AuthCodeTokenExchangeException events.
+    /// </summary>
+    public class AuthCodeTokenExchangeExceptionEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Gets or sets the OidcClient.
+        /// </summary>
+        public IOidcClient OidcClient { get; set; }
```

Comment:
If the client returns an AuthCodeTokenExchangeExceptionEventArgs, then why do you need a reference to the client here?

Reply:
An instance of the ...Args class is passed to the subscriber of the event when the event is triggered, this gives the subscriber access to the state of the client at the time the event handler executes.